### PR TITLE
Set up localization structure

### PR DESCRIPTION
**Pull request by beatrizz**
https://github.com/acjc/heist/issues/57

This sets up the localization structure.

I haven't done the roles yet because we need to modify how we create them to pass the context. I'll do that separately.

We do need to specify 'en' translations even if we assume that English is always going to be the default language. Otherwise, it picks up the Spanish translation when English is my first locale and Spanish my second one.

I'm not translating anything until later on, because the strings are likely to change. I've done one string to check that it works.

I think we should finish this before the rebranding to ghost theme, it'll be easier if all the strings are in the same place.

IMPORTANT: I have had a play with the app but I haven't tested all the combinations, there are too many :S

### CODE REVIEW

File: lib/l10n/intl_messages.arb [hunk 1]
```diff
@@ -0,0 +1,438 @@
+{
```
Comment (beatrizz) [inline lib/l10n/intl_messages.arb]: This was automatically generated

File: lib/l10n/intl_en.arb [hunk 2]
```diff
@@ -0,0 +1,438 @@
+{
```
Comment (beatrizz) [inline lib/l10n/intl_en.arb]: This is a copy of intl_messages, which was automatically generated.

File: lib/l10n/intl_es.arb [hunk 3]
```diff
@@ -0,0 +1,438 @@
+{
```
Comment (beatrizz) [inline lib/l10n/intl_es.arb]: This is a copy of intl_messages, which was automatically generated, where I've translated a few strings to test.

File: lib/l10n/messages_all.dart [hunk 4]
```diff
@@ -0,0 +1,65 @@
+// DO NOT EDIT. This is code generated via package:intl/generate_localized.dart
```
Comment (beatrizz) [inline lib/l10n/messages_all.dart]: This was automatically generated

File: lib/l10n/messages_en.dart [hunk 5]
```diff
@@ -0,0 +1,132 @@
+// DO NOT EDIT. This is code generated via package:intl/generate_localized.dart
```
Comment (beatrizz) [inline lib/l10n/messages_en.dart]: This was automatically generated

File: lib/l10n/messages_es.dart [hunk 6]
```diff
@@ -0,0 +1,132 @@
+// DO NOT EDIT. This is code generated via package:intl/generate_localized.dart
```
Comment (beatrizz) [inline lib/l10n/messages_es.dart]: This was automatically generated

File: lib/l10n/messages_messages.dart [hunk 7]
```diff
@@ -0,0 +1,132 @@
+// DO NOT EDIT. This is code generated via package:intl/generate_localized.dart
```
Comment (beatrizz) [inline lib/l10n/messages_messages.dart]: This was automatically generated

**Comment by acjc** [comment]
I have fixed the dependency problem issue in my branch (it was only a minor one).

**Comment by beatrizz** [comment]
I don't think it was the same issue. The intl stuff requires a different version of the SDK from something else and we can't do flutter upgrade because it'd override some stuff. Trying to fix it, I broke my local env :( I'll give it another go tonight.

**Comment by beatrizz** [comment]
Yay, it now works :) I've had to reinstall flutter (because it didn't let me upgrade it) and then apply all the available updates. I've also cherry-picked your dependencies fix here as I wasn't sure if it'll help. I'll merge your change and the l10n stuff.
Thanks for sorting out the strings conflicts :)

**Pull request merged**